src/h5web/toolbar/Toolbar.module.css

```diff
@@ -91,37 +92,32 @@
 }
 
 .btn:hover > .btnLike {
-  background-color: var(--secondary-lighter);
-  box-shadow: var(--btn-shadow-idle) var(--btn-shadow-color);
-}
-
-.btn:active > .btnLike {
-  box-shadow: var(--btn-shadow-pressed) var(--btn-shadow-color);
+  background-color: var(--h5w-btn-hover--bgColor, whitesmoke);
+  box-shadow: -1px -1px 2px inset var(--h5w-btn-hover--shadowColor, var(--h5w-btnRaised--shadowColor, gray));
 }
 
 .btn[data-raised]:hover > .btnLike {
-  box-shadow: var(--btn-shadow-raised) var(--btn-shadow-color),
-    var(--btn-shadow-idle) var(--btn-shadow-color);
+  box-shadow: -1px -1px 3px inset var(--h5w-btnRaised-hover--shadowColor, dimgray);
 }
 
-.btn[aria-pressed='true'] > .btnLike,
-.btn[aria-checked='true'] > .btnLike {
-  background-color: var(--secondary-light);
-  box-shadow: var(--btn-shadow-pressed) var(--btn-shadow-dark-color);
+.btn:active > .btnLike,
+.btn[data-raised]:active > .btnLike,
+.btn[aria-pressed='true'] > .btnLike, /* toggle buttons, domain slider "edit" button */
+.btn[aria-checked='true'] > .btnLike, /* toggle group buttons */
+.btn[aria-haspopup][aria-expanded='true'] > .btnLike  /* selectors and overflow menu buttons */ {
```

Comment:
Adding `[aria-haspopup]` to the "expanded" selector so that the domain slider's "edit" button doesn't appear as pressed when hovered (as hovering it expands the domain slider's tooltip). Contrary to the overflow menu button and the buttons of the drop-down selectors, the "edit" button doesn't have `aria-haspopup`; only `aria-controls`.